### PR TITLE
Alert agent to look for missing device publishes

### PR DESCRIPTION
**Pull request by mikeroup**
This change is [Reviewable](https://reviewable.io/reviews/volttron/volttron/708)

### CODE REVIEW

File: services/core/AlertAgent/alerter/agent.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+__version__ = '0.1'
+
+
+class AlertAgent(Agent):
+    def __init__(self, config_path, **kwargs):
+        super(AlertAgent, self).__init__(**kwargs)
+        config = utils.load_config(config_path)
+        self.wait_time = {}
+        self.topic_ttl = {}
```
Comment (craig8) [inline services/core/AlertAgent/alerter/agent.py]: What is ttl?  Time to live?

Comment (mikeroup) [inline services/core/AlertAgent/alerter/agent.py]: yes

**Comment by craig8** [comment]
Reviewed 4 of 5 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/volttron/volttron/708)*

**Comment by craig8** [comment]
Good job @mikeroup 

I have included an issue that I would like you to look into as well #710

**Pull request merged**